libs/feature/search/src/lib/state/effects.ts

```diff
@@ -130,17 +130,19 @@ export class SearchEffects {
             }
             return this.filterGeometry$.pipe(
               tap((geom) => {
-                try {
-                  const trace = validGeoJson(geom, true) as string[]
-                  if (trace?.length > 0) {
-                    throw new Error(trace.join('\n'))
+                if (geom) {
```

Comment:
could you maybe invert this statement? e.g. `if (!geom) { return; }`
That would save one level of depth :)